test/groovy/com/sap/piper/ConfigurationHelperTest.groovy

```diff
@@ -420,4 +420,23 @@ class ConfigurationHelperTest {
                                    .use()
     }
 
+    @Test
```

Comment:
Adding these tests here is beneficial.

Would be cool to have also a corresponding GString test here. I mean having a corresponding GString test on the level of neoDeploy (in that context explicitly for the deployMode) is not wrong, but basically the bug was located here in the helper method. Hence it would be IMO preferable to have also a the GString test here. The way between the test code and the code under test is shorter ...